ethcore/src/miner/miner.rs

```diff
@@ -1018,6 +1018,15 @@ impl miner::MinerService for Miner {
 		-> Vec<Arc<VerifiedTransaction>>
 	where
 		C: ChainInfo + Nonce + Sync,
+	{
+		// No special filtering options applied (neither tx_hash, receiver or sender)
+		self.ready_transactions_filtered(chain, max_len, None, None, None, ordering)
+	}
+
+	fn ready_transactions_filtered<C>(&self, chain: &C, max_len: usize, tx_hash: Option<H256>, receiver: Option<H160>, sender: Option<H160>, ordering: miner::PendingOrdering)
```

Comment:
I included the `tx_hash` since it was part of the originally requested issue. But yes; `eth_getTransactionByHash` does the same thing, assuming it also works with pending transactions and not fully completed/verified ones (?) I'll take a deeper look at the code.